packages/idealo.design/src/ui/pages/BlogPage/Editor/Editor.jsx

```diff
@@ -163,8 +171,28 @@ class RichTextEditor extends React.Component {
 
   handleSubmit(e){
     e.preventDefault();
+    if(!this.handleValidation() && this.state.error['title-value']){
+      alert("Title is invalid. Please choose a different title!");
+      return;
+    }
+
+    if(!this.handleValidation() && this.state.error['title-empty']){
+      alert("Please choose a title!");
+      return;
+    }
+
+    if(!this.handleValidation() && this.state.error['categoryDisplayValue']){
+      alert("Please choose a category!");
+      return;
+    }
+
+    if(!this.handleValidation() && this.state.error['editorState']){
+      alert("Please write something!");
+      return;
+    }
+
```

Comment:
where is the difference between yourPR and Iuliias PR (https://github.com/idealo/nwp/pull/66) ?
She also checked that forms.
So I thought your task was, to check only the title's name...

Reply:
okay. thanks for explanation :-)